modules/local/parse_kraken2report.nf

```diff
@@ -122,7 +122,12 @@ kraken_dict = kraken_taxonomy2hierarchy(kraken_list)
 list_kraken = get_all_keys(kraken_dict)
 result = remove_incomplete_taxa(list_kraken)
 result = generate_dict_for_lookup(result)
-result_to_filter = result["$params.tax2filter"] + ["$params.tax2filter"]
+try:
```

Comment:
In the whole file seems to be something off with the indentation I think. One indent missing?